build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/CreateAndroidEmulator.cs

```diff
@@ -92,9 +92,11 @@ void Run (string android)
 			if (Log.HasLoggedErrors)
 				return;
 
+			string fullSystemImagePath = Path.Combine (AndroidSdkHome, "system-images", $"android-{SdkVersion}", "default", AndroidAbi);
 			File.AppendAllLines (configPath, new string[] {
 				$"disk.dataPartition.size={diskSize}M",
-				$"hw.ramSize={ramSize}"
+				$"hw.ramSize={ramSize}",
+				$"image.sysdir.1={fullSystemImagePath}"
```

Comment:
Within an array/collection/etc. context, please end every line with `,`, so that when we need to add a line in the future we minimize "extra line noise" (as is the case here, with `hw.ramSize`).